app/assets/stylesheets/components/_footer_nav.scss

```diff
@@ -0,0 +1,23 @@
+.footer {
+  background-color: #FF2840;
```

Comment:
Background-colors have been specified in the colors.scss file please make sure to use them here instead

Reply:
done